django/contrib/staticfiles/storage.py

```diff
@@ -57,7 +57,10 @@ class HashedFilesMixin:
                     """@import url("%(url)s")""",
                 ),
                 (
-                    r"(?m)(?P<matched>)^(/\*# (?-i:sourceMappingURL)=(?P<url>.*) \*/)$",
+                    (
+                        r"(?m)(?P<matched>)^(/\*#[ \t]"
+                        r"(?-i:sourceMappingURL)=(?P<url>.*)[ \t]*\*/)$"
```

Comment:
We should add a test for source map comments with leading and trailing tabs.

Reply:
Added. Test fails on `main`.